pom.xml

```diff
@@ -159,6 +160,12 @@
             <version>4.1.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>1.9</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do we need this?
If it's just there for overriding a vulnerable one, please explain that in a comment.

Reply:
removed it